Automattic/newspack-theme
# Minify styles

### All Submissions:






### Changes proposed in this Pull Request:



Adds CSS minification to the created stylesheets.

Also fixes a warning triggered by Sass about rule nesting. Nested rules should be placed after other rules.

### How to test the changes in this Pull Request:

1. Run `npm run build` and observe the `<theme-slug>/style.css` files are minified
2. Smoke-check the front-end to see if it looks as expected

### Other information:
